### DISCUSSION TITLE
how do i run commands in github im dumb

### DISCUSSION THREAD

**Discussion by LordEvergreen** [QuiteAFancyEmerald/InvisiProxy#183] [Q&A] [answered]
how do i run commands in github im dumb

**Comment by nanospearing** [comment]
What do you mean? I dont think you can run commands in github

**Comment by foxsouns** [comment, marked as answer]
if you wanna be stupid and recursive, and you're stuck on browser, you _could_ use replit to interface with github. not saying you should, or it's useful, as most of `git` is accessible via actually using github, but it's an option.
